grails-app/domain/org/pih/warehouse/inventory/CycleCountItem.groovy

```diff
@@ -53,7 +53,7 @@ class CycleCountItem implements Comparable {
                 facility: facility.toBaseJson(),
                 product: product,
                 inventoryItem: inventoryItem,
-                binLocation: location?.toBaseJson(),
+                binLocation: location?.toJson(),
```

Comment:
I think it would be better to add zone name and zone id to the `toBaseJson()` instead using `toJson`, because it checks some activity codes, etc, which might degrade performance in huge cycle counts.